src/Form/index.tsx

```diff
@@ -683,16 +684,22 @@ function Form({
         // @ts-expect-error TS(2345): Argument of type 'Promise<any[]>' is not assignabl... Remove this comment to see the full error message
         .fetchSession(formPromise, true)
         .then(([session, steps]) => {
+          initState.sessions[formName] = session;
```

Comment:
i think you should move this line into the fetchSession function. then you can also remove this line https://github.com/feathery-org/feathery-react/blob/master/src/utils/init.ts#L167